src/main/java/com/solvd/micro9/users/service/SequenceGeneratorService.java

```diff
@@ -0,0 +1,9 @@
+package com.solvd.micro9.users.service;
+
+import java.util.concurrent.ExecutionException;
+
+public interface SequenceGeneratorService {
+
+    Long generateSequence(final String sequenceName) throws InterruptedException, ExecutionException;
```

Comment:
@eyakauleva, JFYI: This logic should be represented on MongoDB side: https://www.mongodb.com/basics/mongodb-auto-increment